Lib/fontTools/designspaceLib/__init__.py

```diff
@@ -540,6 +597,11 @@ def _addSource(self, sourceObject):
         sourceElement.append(locationElement)
         self.root.findall('.sources')[0].append(sourceElement)
 
+    def _addLib(self, dict):
+        libElement = ET.Element('lib')
+        libElement.append(to_plist(dict))
```

Comment:
even if it's just in a nested method scope like in here, I like to avoid shadowing built-in types.
makes it harder to read as the syntax highlighting thinks it's a built-in dict type, whereas this is actually a local variable.